cli.js

```diff
@@ -27,19 +36,34 @@ const cli = meow(`
 
 	Options
 	  --copy -c  Copy the first emoji to the clipboard
+	  --skin-tone -s  Set the default skin tone of the emoji (0 to 5)
 
 	Run it without arguments to enter the live search
+	Use Up/Down keys during live search to change the skin tones
 `, {
 	boolean: [
 		'copy'
 	],
 	alias: {
-		c: 'copy'
+		c: 'copy',
+		s: 'skinTone'
 	}
 });
 
+if (typeof cli.flags.skinTone === 'number') {
```

Comment:
Actually, without validation `Number(cli.flags.skinTone)` will return `NaN`, which makes the Math.max/min functions always return `NaN` as well, which is not an acceptable `skinNumber`.

We could do `Number(cli.flags.skinTone) | 0`, but this would make the default setting change to 0 when an invalid input was used for the flag (such as a string), which I think may not be the desired behaviour...

So I think it's still better to validate the flag as number first, ignoring any other input types, while still coercing the `skinNumber` within the interval.